b2sdk/sync/policy.py

```diff
@@ -29,7 +29,28 @@ class AbstractFileSyncPolicy(object):
     DESTINATION_PREFIX = NotImplemented
     SOURCE_PREFIX = NotImplemented
 
-    def __init__(self, source_file, source_folder, dest_file, dest_folder, now_millis, args):
+    NEWER_FILE_SKIP = 101
+    NEWER_FILE_REPLACE = 102
+
+    NEWER_FILE_MODES = [NEWER_FILE_REPLACE, NEWER_FILE_SKIP]
+
+    COMPARE_VERSION_MODTIME = 201
+    COMPARE_VERSION_SIZE = 202
+    COMPARE_VERSION_NONE = 203
+
+    COMPARE_VERSION_MODES = [COMPARE_VERSION_MODTIME, COMPARE_VERSION_SIZE]
```

Comment:
this is a bit messy. I think we should use https://docs.python.org/3.4/library/enum.html#ensuring-unique-enumeration-values (and https://pypi.org/project/enum34/ on 2.7 - we already have conditional installation for something)

Reply:
@ppolewicz I don't see any conditional installation for enum for 2.7.